zephyr/Kconfig

```diff
@@ -1,3 +1,90 @@
-# Copyright 2022 Espressif Systems (Shanghai) PTE LTD
+# Copyright (c) 2024 Espressif Systems (Shanghai) Co., Ltd.
+# SPDX-License-Identifier: Apache-2.0
 
-# This file is required by Zephyr's build system and can be empty
+if MCUBOOT
+
+config EFUSE_VIRTUAL_OFFSET
+	hex
+	default 0x250000 if SOC_SERIES_ESP32
+	default 0x250000 if SOC_SERIES_ESP32S2
+	default 0x250000 if SOC_SERIES_ESP32S3
+	default 0x250000 if SOC_SERIES_ESP32C3
+	default 0x250000 if SOC_SERIES_ESP32C6
+
+config EFUSE_VIRTUAL_SIZE
+	hex
+	default 0x2000 if SOC_SERIES_ESP32
+	default 0x2000 if SOC_SERIES_ESP32S2
+	default 0x2000 if SOC_SERIES_ESP32S3
+	default 0x2000 if SOC_SERIES_ESP32C3
+	default 0x2000 if SOC_SERIES_ESP32C6
+
+config EFUSE_MAX_BLK_LEN
+	int
+	default 192 if SOC_SERIES_ESP32
+	default 256 if SOC_SERIES_ESP32S2
+	default 256 if SOC_SERIES_ESP32S3
+	default 256 if SOC_SERIES_ESP32C3
+	default 256 if SOC_SERIES_ESP32C6
+
+#config SPI_FLASH_ROM_DRIVER_PATCH #remove? (it's also defined on zephyr/soc/espressif/commom/Kconfig.flash)
```

Comment:
I think it indeed could be hidden here as it is not something user can/should change.

Reply:
But it needs to be moved outside `MCUBOOT` sesction.

Reply:
I'll remove it, since it is defined on zephyr/soc/espressif/commom/Kconfig.flash with default to y (independent of SOC_SERIES and bootloader option)